README.md

```diff
@@ -81,6 +81,10 @@ Repeatable::Expression::Difference.new(included: expression, excluded: another_e
 
 # DATES
 
+# Every day
+{ daily: {} }
```

Comment:
There's something about this empty hash feels a bit funny, but it's also fine 🤷🏼‍♂️